apps/web/pages/[user]/calendar-cache/[month].tsx

```diff
@@ -29,16 +27,17 @@ export const getStaticProps: GetStaticProps<
       selectedCalendars: true,
     },
   });
-  const startDate = (
-    dayjs(month, "YYYY-MM").isSame(dayjs(), "month") ? dayjs.utc() : dayjs.utc(month, "YYYY-MM")
-  ).startOf("day");
-  const endDate = startDate.endOf("month");
+  // Subtract 11 hours from the start date to avoid problems in UTC- time zones.
```

Comment:
Are there tests around this? If not, we should add some because this logic seems tricky.

Reply:
@keithwillcode  Sorry, is an unralated change, this is from this [PR](https://github.com/calcom/cal.com/pull/9282), but yeah you right, I will add a test in the other PR, The time added is based on this https://www.timeanddate.com/time/map/
![imagen](https://github.com/calcom/cal.com/assets/525369/c510f8c5-7c00-4471-98c7-36d0a4815c59)